### DISCUSSION TITLE
Temp value cutoff scaling

### DISCUSSION THREAD

**Discussion by SquarishRectangle** [LeelaChessZero/lc0#1654] [Ideas]
Temp value cutoff (TVC) is a great param but it is based on winrate which scales very weirdly. Even only at 5, when she is winning easily or losing badly, she will blunder a queen or both rooks without batting an eyelash. 

I propose a new parameter called temp value cutoff scaling: Type float, min 0.00, max 1.00, default 1.00.

If this param is set to anything but 1 it will decrease TVC when lc0 is losing. The lower the value the lower TVC scales. 

For example, if TVC is set to 10 and TVC scaling is set to 0.5, Lc0 will have TVC of 5 when losing, TVC of 7.5 when the position is equal, and TVC of 10 when winning. 

This is a simple change but I believe it will help very much with both Training data generation and Human play.

**Comment by mooskagh** [comment]
It's better to propose ideas in our Discord chat at lc0.org/chat, that way more people will notice it.
We also have github discussions, which I believe noone looks into anyway, but it's still a better place to discuss ideas than issues. I'm moving this to the discussions.
